### DISCUSSION TITLE
"Get Boilerplate" not working on XAMPP

### DISCUSSION THREAD

**Discussion by LordRazen** [joomengine/Joomla-Component-Builder#669] [Q&A]
Hi, 

I start writing a plugin for my component and would like to get the boilerplates from the "Get Boilerplate" Button inside the Plugins area. However: It doesnt work on XAMPP 3.2.4. It throws the message "Getting joomla_plugin boilerplate was successfully!", but methods and properties are still empty. 

I run the boilerplate button on another test environment on a webserver and it worked, but when I exported / imported the methods and properties, not all were important for unknown reasons. Only 20 from 39 properties and 90/120 methods made the export, the rest vanished somehow

I found an old thread inside the old Google forum where CURL was requested in order to make it work. I checked with phpinfo() - it's activated. 

What else could cause the problem? Having the boilerplates in XAMPP would be very useful indeed. 

Thanks for the help ^^

**Comment by Llewellynvdm** [comment]
Woops... missed this question, I have no idea, have you tried using the [bitnami stack](https://www.youtube.com/watch?v=zpS52k89YcI&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=72) instead of the XAMPP?

I am all linux... and have not used Mac or MS for years really (except in VM's), so I can't really be of help to you on this.

**Reply by CStefanSv** [reply]
I have this problem also but running on WAMP. 
Reading the WAMP-logfiles I found this:
PHP Warning:  fopen(/D:\wamp64\www\PT64\administrator/components/com_componentbuilder/816bebd2b207dc25fd9d33895cb7bcb4.md): failed to open stream: No such file or directory i
which to me looks peculiar with both slash and backslash in the foldername.
Having the debuggning system activated on Joomla I can see that the boilerplate codes saeems to be there but cant not be stored. I have tried to find out where the foldername is created but have not found the place.
Stefan
